policyengine_us/reforms/congress/tlaib/income_security_package/income_security_package.py

```diff
@@ -0,0 +1,412 @@
+from policyengine_us.model_api import *
+
+
+def create_income_security_package() -> Reform:
+    class baby_bonus(Variable):
+        value_type = float
+        entity = Person
+        definition_period = YEAR
+        label = "Baby Bonus Act payment"
+        unit = USD
+        documentation = (
+            "One-time payment for newborns under the Baby Bonus Act"
+        )
+        reference = "placeholder - bill not yet introduced"
+
+        def formula(person, period, parameters):
+            p = parameters(
+                period
+            ).gov.contrib.congress.tlaib.income_security_package.baby_bonus_act
+
+            age = person("age", period)
+            max_age = p.max_child_age
+            is_eligible_child = age < max_age
+
+            amount = p.amount
+
+            return is_eligible_child * amount
+
+    class boost_payment(Variable):
+        value_type = float
+        entity = Person
+        definition_period = YEAR
+        label = "BOOST Act payment"
+        unit = USD
+        documentation = (
+            "Monthly payments under the BOOST Act for eligible adults"
+        )
+        reference = "placeholder - bill not yet introduced"
+
+        def formula(person, period, parameters):
+            p = parameters(
+                period
+            ).gov.contrib.congress.tlaib.income_security_package.boost_act
+
+            age = person("age", period)
+            min_age = p.min_age
+            max_age = p.max_age
+
+            is_eligible = (age >= min_age) & (age <= max_age)
+
+            monthly_amount = p.amount
+            annual_amount = monthly_amount * MONTHS_IN_YEAR
+
+            return is_eligible * annual_amount
```

Comment:
```suggestion
            is_eligible = (age >= p.min_age) & (age <= p.max_age)

            annual_amount = p.amount * MONTHS_IN_YEAR

            return is_eligible * annual_amount
```

We don't need to define parameters individually if they are no a `.calc` or breakdown operation or if we only use it once

Reply:
same for other variables